docs/source/cli.mdx

```diff
@@ -49,5 +49,14 @@ Note that you should pass the `--trust_remote_code` argument only if you trust t
 
 For example:
 ```bash
->>> datasets-cli convert_to_parquet USERNAME/DATASET_NAME --token USER_ACCESS_TOKEN
+>>> datasets-cli convert_to_parquet USERNAME/DATASET_NAME
 ```
+
+<Tip>
+
+Do not forget that you need to log in first to your Hugging Face account:
+```bash
+huggingface-cli login
```

Comment:
no `>>>` here? 😛

Reply:
There is only input (no output) to be copied/pasted. But I could add it as well, so it is aligned everywhere...

Reply:
Done.